engine/classes/Elgg/Traits/Database/LegacyQueryOptionsAdapter.php

```diff
@@ -1135,30 +1075,14 @@ protected function normalizeGroupByClauses(array $options = []) {
 		}
 
 		if (is_string($options['group_by'])) {
-			$clause = $options['group_by'];
-
-			$options['group_by'] = explode(',', $options['group_by']);
-
-			if (count($options['group_by']) > 1) {
-				elgg_deprecated_notice("
-					Using literal MySQL statements in 'group_by' options parameter is deprecated.
-					Instead use a closure that receives an instanceof of QueryBuilder
-					and returns a prepared clause.
-					
-					{{ $clause }}
-				", '3.0');
-			}
+			$options['group_by'] = (array) trim($options['group_by']);
```

Comment:
did you check the function docs of ``elgg_get_entities`` for correctness

Reply:
yep... string support is undocumented